src/pages/Game/components/Square/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import './styles.scss'
+import PlanetIcon from '../PlanetIcon'
+
+
+const Square =
+  ({ value, onClick }) => (
+    <button
+      className="square"
+      onClick={onClick}
+    >
+      {<PlanetIcon player={value} />}
+    </button>
+  )
```

Comment:
The word `value` for a Square is not that clear to me of what represents. Instead using `player` would let me know better that a Square can be "owned" or "marked" by a player (always thinking that we are implementing a Tic Tac Toe game).

Also, you can resolve right here the value of the Planet Icon by evaluating who is the player, and then render that `<img>` tag directly.

```suggestion
const Square =
  ({ player, onClick }) => (
    <button
      className="square"
      onClick={onClick}
    >
      <img src={getPlayerIcon(player)} alt={player}/>
    </button>
  )
  
const getPlayerIcon(player) => player === 'X' ? saturn : earth
```

This is a small refactor suggestion that saves you the need to implement a PlanetIcon component. If you then consider taking another step in the refactoring process, think about the possibility of getting the player icon from the player itself. That way this Square component would have one less responsibility (no need to do this right now).